Plugin/Model/SavePlugin.php

```diff
@@ -79,4 +80,31 @@ public function afterSave(PageRepository $subject, Page $result)
 
         return $result;
     }
+
+    private function buildUrlFromPage(Page $page): string
+    {
+        if (!array_key_exists(0, $page->getStoreId())) {
```

Comment:
Is this check still necessary since line 92?

Reply:
Whoops! That was messy, my bad. I've fixed it in a new commit.